### ISSUE TITLE
cubefromshape fails with Shapefile.jl version 0.6

### ISSUE THREAD

**Issue by felixcremer** [esa-esdl/ESDL.jl#138]
With the new Shapefile version I get the following  error, when I try to load a shapefile into ESDL.
The loading of the shapefile works as expected, when I set the Shapefile version to 0.5
I am not sure, whether this is due to my shapefile. 
If it would help you tracking down the error, I can send you the shapefile and the cube with which I get the error.
```julia
LoadError: MethodError: no method matching rasterizepoly!(::Array{Int32,2}, ::Array{Union{Missing, Shapefile.Polygon},1}, ::NamedTuple{(:left, :right, :top, :bottom),NTuple{4,Float64}})
Closest candidates are:
  rasterizepoly!(::Any, !Matched::Array{#s85,1} where #s85<:GeoInterface.AbstractMultiPolygon, ::Any) at /home/crem_fe/.julia/dev/ESDL/src/Proc/Shapes.jl:137
  rasterizepoly!(::Any, !Matched::Array{T<:GeoInterface.AbstractPoint,1}, ::Any; value, wrap) where T<:GeoInterface.AbstractPoint at /home/crem_fe/.julia/dev/ESDL/src/Proc/Shapes.jl:144
  rasterizepoly!(::Any, !Matched::Shapefile.Polygon, ::Any; value) at /home/crem_fe/.julia/dev/ESDL/src/Proc/Shapes.jl:181
in expression starting at /home/crem_fe/Documents/rqa_deforestation/scripts/plot_rocs.jl:20
#rasterize!#27(::NamedTuple{(:left, :right, :top, :bottom),NTuple{4,Float64}}, ::Nothing, ::typeof(ESDL.Proc.rasterize!), ::Array{Int32,2}, ::String) at Shapes.jl:130
(::getfield(ESDL.Proc, Symbol("#kw##rasterize!")))(::NamedTuple{(:bb,),Tuple{NamedTuple{(:left, :right, :top, :bottom),NTuple{4,Float64}}}}, ::typeof(ESDL.Proc.rasterize!), ::Array{Int32,2}, ::String) at none:0
#cubefromshape_single#20(::Nothing, ::Type, ::typeof(ESDL.Proc.cubefromshape_single), ::String, ::RangeAxis{Float64,:Lon,StepRangeLen{Float64,Base.TwicePrecision{Float64},Base.TwicePrecision{Float64}}}, ::RangeAxis{Float64,:Lat,StepRangeLen{Float64,Base.TwicePrecision{Float64},Base.TwicePrecision{Float64}}}) at Shapes.jl:85
cubefromshape_single at Shapes.jl:81 [inlined]
#cubefromshape#22 at Shapes.jl:106 [inlined]
cubefromshape(::String, ::RangeAxis{Float64,:Lon,StepRangeLen{Float64,Base.TwicePrecision{Float64},Base.TwicePrecision{Float64}}}, ::RangeAxis{Float64,:Lat,StepRangeLen{Float64,Base.TwicePrecision{Float64},Base.TwicePrecision{Float64}}}) at Shapes.jl:105
#cubefromshape#21(::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::typeof(cubefromshape), ::String, ::ESDL.Cubes.ESDLZarr.ZArrayCube{Union{Missing, Float32},3,Zarr.ZArray{Union{Missing, Float32},3,Zarr.NoCompressor,Zarr.DirectoryStore},Nothing}) at Shapes.jl:103
cubefromshape at Shapes.jl:103 [inlined]
plot_roc(::ESDL.Cubes.ESDLZarr.ZArrayCube{Union{Missing, Float32},3,Zarr.ZArray{Union{Missing, Float32},3,Zarr.NoCompressor,Zarr.DirectoryStore},Nothing}, ::String, ::String, ::String, ::String, ::Symbol, ::Bool) at roc_plotting.jl:14
plot_roc(::ESDL.Cubes.ESDLZarr.ZArrayCube{Union{Missing, Float32},3,Zarr.ZArray{Union{Missing, Float32},3,Zarr.NoCompressor,Zarr.DirectoryStore},Nothing}, ::String, ::String, ::String, ::String, ::Symbol) at roc_plotting.jl:14
top-level scope at plot_rocs.jl:20
include at boot.jl:328 [inlined]
include_relative(::Module, ::String) at loading.jl:1094
include(::Module, ::String) at Base.jl:31
include(::String) at client.jl:431
top-level scope at base.jl:42
```

**Comment by meggart** [comment]
That was cause by a signature change in the Shapefile package. This change here fixes it https://github.com/esa-esdl/ESDL.jl/commit/d148379a918f79ff4fff74283cc80a4eeae19604#diff-aa0aaccb5d5a9781a90d74897396caedL136 or you just check out the latest threads branch.

**Comment by felixcremer** [comment]
This is fixed in #137
